app/views/manage_competencies/caseworker_competencies/_table_cells.html.erb

```diff
@@ -1,7 +1,12 @@
 <td class="govuk-table__cell">
-  <% if Allocating.user_competencies(caseworker.id).blank? %>
-      <%= t('.no_competencies') %>
-  <% else %>
-    <%= Allocating.user_competencies(caseworker.id).map(&:humanize).join(', ') %>
-  <% end %>
+  <a class="govuk-link govuk-link--no-visited-state" href="<%=edit_manage_competencies_caseworker_competency_path(id: caseworker)%>">
+    <% if Allocating.user_competencies(caseworker.id).blank? %>
+        <%= t('.no_competencies') %>
+    <% else %>
+      <%= Allocating.user_competencies(caseworker.id).map(&:humanize).join(', ') %>
```

Comment:
If we do add a presenter then perhaps we could wrap this in a mothod which explaines it like `formatted_user_competencies` or something similar